source/ocsp-support/tests/README.rst

```diff
@@ -85,6 +88,9 @@ extra cases and may help reveal unexpected behavior.
 library’s implementation of OCSP will need to document these failures as
 described under `Documentation
 Requirements <../ocsp-support.rst#documentation-requirements>`__
+Additionally, drivers that fail the "Soft Fail" will need to change any applicable defaults as described under
```

Comment:
Should this be "Soft Fail Test", as written in the spec document? As-is, "fail the soft fail" reads oddly.

Reply:
Fixed!